OXRS-SHA-StateMonitor-ESP32-FW.ino

```diff
@@ -100,23 +99,25 @@ void setup()
   // initialize screen
   screen.begin();
   
-  // Speed up I2C clock for faster scan rate (after bus scan and screen init)
+  // Speed up I2C clock for faster scan rate (after bus scan)
   Wire.setClock(I2C_CLOCK_SPEED);
 
-  // Display the firmware version and initialise the port display
-  screen.draw_logo(FW_VERSION);
+  // Display the header and initialise the port display
+  screen.draw_header(FW_MAKER_CODE, FW_SHORT_NAME, FW_VERSION, FW_PLATFORM);
   screen.draw_ports(g_mcps_found);
 
   // Set up ethernet and obtain an IP address
   byte mac[6];
   initialiseEthernet(mac);
+  
+  // Display IP and MAC addresses on screen
+  screen.show_ethernet();
 
   // Set up connection to MQTT broker
   initialiseMqtt(mac);
 
-  // Display IP and MAC addresses on screen
-  screen.show_IP(Ethernet.localIP());
-  screen.show_MAC(mac);
+  // Display temperature on screen
+  screen.show_temp(12.3456);               // for test now. value will be replaced by measured tempereture

```

Comment:
I know this is temporary but it should probably be in `loop()` and on a timer so we only update every 60s or something?

Reply:
prepared `loop() ` with `update_temperature();` and created skeleton function